src/Components/CreateImageWizardV2/steps/Review/Footer/CreateDropdown.tsx

```diff
@@ -41,14 +42,41 @@ const CreateDropdown = ({
 
   return (
     <DropdownList>
-      <DropdownItem onClick={onSave} ouiaId="wizard-create-save-btn">
-        Save changes
-      </DropdownItem>
       <DropdownItem onClick={onSaveAndBuild} ouiaId="wizard-create-build-btn">
         Save and build images
       </DropdownItem>
     </DropdownList>
   );
 };
 
-export default CreateDropdown;
+export const CreateSaveButton = ({
+  setIsOpen,
+  getBlueprintPayload,
+}: CreateDropdownProps) => {
+  const [createBlueprint, { isLoading }] = useCreateBlueprintMutation({
+    fixedCacheKey: 'createBlueprintKey',
+  });
+  const onSave = async () => {
+    const requestBody = await getBlueprintPayload();
+    setIsOpen(false);
+    requestBody && createBlueprint({ createBlueprintRequest: requestBody });
+  };
+  return (
+    <MenuToggleAction onClick={onSave} id="wizard-create-save-btn">
```

Comment:
Looks good! Since there's an ouiaId removed - @jrusz Does pr_check use `ouiaId="wizard-create-save-btn"` anywhere?

Reply:
pr_check doesn't run the blueprints test yet and I don't use it yet in my V2 wizard update MR so it's all good :)

Reply:
Got it, so we can add it when/if needed. Thank you! :)

Reply:
Yeah I don't need ouiaID for the actual dropdown items, I'd only use it to locate the Dropdown for example, the items are then accessed by their actual names/values.